src/neo4j_genai/retrievers/vector.py

```diff
@@ -65,6 +65,7 @@ class VectorRetriever(Retriever):
         index_name (str): Vector index name.
         embedder (Optional[Embedder]): Embedder object to embed query text.
         return_properties (Optional[list[str]]): List of node properties to return.
+        result_formatter (Optional[Callable[[neo4j.Record], RetrieverResultItem]]): Provided custom function to transform a neo4j.Record to a RetrieverResultItem.
```

Comment:
Maybe we should add a note about the input for this function, ie what is inside the record? (i.e. node and score for the vector retriever, where node is a dict (?))

Reply:
[Added here](https://github.com/neo4j/neo4j-genai-python/pull/73/commits/46c3b961aba6ea5ee0487fc951a2ab0a064380b1)